project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.2
```

Comment:
This dictates the minimum sbt version for the plugin, right?
Maybe stay on something lower than latest, e.g. 1.5.0?

Reply:
So I think that the reason sbt's inbuilt graph checking tool is all private is to actually not conflict with sbt-dependency-graph so lets see if it builds without problems.

Reply:
Okay so it seems to work fine even with the old plugin